src/helpers/get-cron-time.ts

```diff
@@ -0,0 +1,12 @@
+export const getCronTime = (cron: string): string => {
```

Comment:
use object instead of switch so we can use `keyof` on it to make sure we always access proper keys (and never throw)